content/learn/book/welcome/app/_index.md

```diff
@@ -0,0 +1,82 @@
++++
+title = "Creating Bevy apps"
+weight = 2
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+Bevy programs store and execute all of their game logic and data with a single `App` data structure.
+Let's make a trivial Hello World app.
```

Comment:
This seems like it should precede the first line.

I would like to see the flow if "Bevy programs store and execute all of their game logic and data with a single `App` data structure." was moved to below the code block, so the reader know's we're making a trivial game, sees the code, then sees the explanation for said code.

Reply:
I'd like to go explanation -> example in the book; the quick start guides will be better suited for readers who prefer the opposite direction.